test/integration/concurrency_controls_test.rb

```diff
@@ -183,7 +183,46 @@ class ConcurrencyControlsTest < ActiveSupport::TestCase
     assert job.reload.ready?
   end
 
+  if ActiveRecord::Base.connection.adapter_name == "PostgreSQL"
```

Comment:
Since we have `TARGET_DB`, I think I'd use that here, like [here](https://github.com/rails/solid_queue/blob/f0ca6b43b5cefc97b40e3154be6ee535d77b8450/test/models/solid_queue/job_test.rb#L257-L267).

Reply:
That is better.   I'll submit the change.